jetbrains-core/src/software/aws/toolkits/jetbrains/services/codewhisperer/util/CodeWhispererFileCrawler.kt

```diff
@@ -56,6 +56,13 @@ abstract class CodeWhispererFileCrawler : FileCrawler {
     abstract val fileExtension: String
     abstract val testFilenamePattern: Regex
 
+    /**
+     * For [LocalFileSystem](implementation of virfual file system), the path will be an absolute file path with file separator characters replaced
+     * by forward slash "/"
+     * @see [VirtualFile.getPath]
+     */
+    protected val fileSeperator: String = "/"
```

Comment:
I think there's already a `Files.separator` for this, and that one takes into account different platforms.

Reply:
yes correct but VirtualFile only use `/`

Reply:
I think I should even remove seperator as an method argument like this https://github.com/aws/aws-toolkit-vscode/pull/3638/files#diff-0a6ce5c680e7aeac6f113d229eac60ce065f4eea9549383b85265a8f5072ffd2R166

Reply:
it doesn't need to be there since the platform already normalize it

Reply:
Is this still needed?

Reply:
refactored  and remove the seperator form the arg.

Reply:
was thinking to leave it as it is

Reply:
refer to this commit https://github.com/aws/aws-toolkit-jetbrains/pull/3744/commits/5bb0467de2372f8cfba7538688d05aaa160cd5d7